client/src/components/Messaging/EmojiRow/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from './Style.module.css';
+import Emoji from '../Emoji';
+import '../Emoji/index.js';
+
+const EmojiRow = ({ text, setText }) => {
+  return (
+    <div className={styles.emojiRow}>
```

Comment:
I would prefer to do it with a loop. for example: 
```
const  useEmoji = (symbol)=> {
    setText(text + symbol);
}
const emojis = ["🙂", "😀", "😂"];
emojis.map((sym, index)=>(<Emoji key={index} symbol={sym} onClick={()=>useEmoji(sym)} />))
```

Reply:
Can i call hooks inside loops?